### PR TITLE
Revert "use the Oracle PList format if the JVM Version is at least 1.7"

### PR DESCRIPTION
**Pull request by tofi86**
Reverts UltraMixer/JarBundler#8

See discussion in #8 and #12 for more details.

cc @Vampire

I could merge it by myself as I'm assigned a `Collaborator` role in this repo, but I'd like to get at least a :+1: from the Ultramixer guys :-)

### CODE REVIEW

File: core/src/main/java/com/ultramixer/jarbundler/PropertyListWriter.java [hunk 1]
```diff
@@ -277,133 +287,90 @@ private void buildDOM()
...
 
-        // Java / JavaX entries in the plist dictionary
-        if (bundleProperties.getJavaVersion() < 1.7)
-        {
-            // Apple Java Version
-            writeKey(bundleProperties.getJavaXKey() ? "JavaX" : "Java", dict);
-            Node javaDict = createNode("dict", dict);
+		// Java / JavaX entry in the plist dictionary
+		writeKey(bundleProperties.getJavaXKey() ? "JavaX" : "Java", dict);
+        Node javaDict = createNode("dict", dict);
```
Comment (slajar) [inline core/src/main/java/com/ultramixer/jarbundler/PropertyListWriter.java:292]: code formatting.

File: core/src/main/java/com/ultramixer/jarbundler/PropertyListWriter.java [hunk 2]
```diff
@@ -277,133 +287,90 @@ private void buildDOM()
...
-            // Main class, required
-            writeKeyStringPair("MainClass", bundleProperties.getMainClass(), javaDict);
-
-            // Target JVM version, optional but recommended
-            if (bundleProperties.getJVMVersion() != null)
-            {
-                writeKeyStringPair("JVMVersion", bundleProperties.getJVMVersion(), javaDict);
-            }
+        // Main class, required
+        writeKeyStringPair("MainClass", bundleProperties.getMainClass(), javaDict);
```
Comment (slajar) [inline core/src/main/java/com/ultramixer/jarbundler/PropertyListWriter.java:295]: code formating

File: core/src/main/java/com/ultramixer/jarbundler/PropertyListWriter.java [hunk 3]
```diff
@@ -277,133 +287,90 @@ private void buildDOM()
...
-            // New in JarBundler 2.2.0; Tobias Bley ---------------------------------
+        // Target JVM version, optional but recommended
+        if (bundleProperties.getJVMVersion() != null)
+        {
+            writeKeyStringPair("JVMVersion", bundleProperties.getJVMVersion(), javaDict);
+        }
 
-            // JVMArchs, optional
-            List jvmArchs = bundleProperties.getJVMArchs();
+        // New in JarBundler 2.2.0; Tobias Bley ---------------------------------
```
Comment (slajar) [inline core/src/main/java/com/ultramixer/jarbundler/PropertyListWriter.java:303]: code formating (I assume you are not using spaces?)

**Review by slajar — COMMENTED** [review]
besides the comments the PR looks very good. Good job!

**Comment by tofi86** [comment]
To be honest, I just "git revert"ed the changes by @Vampire 😅 so these formatting issues were part of the original source code ;-)

But once this is merged, I cann fix it together with the other issues I assigned to myself yesterday...

**Comment by tofi86** [comment]
Thanks for the merge!

> Code formating (I assume you are not using spaces?)

@slajar What's your preference? Tabs or spaces? 😏

The code is pretty mixed up, but I'll reformat everything and add an `.editorconfig` file once we have settled on tabs or spaces ;-)

`Jarbundler.java` is mainly tabs, with a few spaces. `PropertyListWriter.java` is spaces only. `JavaProperty.java` is tabs only. I think tabs is the majority in this project, but I don't have any preference ;-)

**Comment by slajar** [comment]
spaces would be great :)

**Comment by tofi86** [comment]
Spaces it is, then! 👍

**Pull request merged**